sig-etcd/annual-report-2023.md

```diff
@@ -4,66 +4,81 @@
+- Released the first alpha for raft `v3.6.0`, a critical roadmap item for etcd `v3.6.0` which adds support for asynchronous storage writes and a number of other changes.
+- Donated `jpbetz/auger` to `etcd-io/auger` with subproject leads in place. With a long term home the project is now revived and being actively maintained.
+- In December as part of etcd `v3.5.11` we introduced new livez/readyz HTTP endpoints following design via the KEP process. The new endpoints are Kubernetes API compliant and a significant improvement over the old health endpoint. These new probes are planned to be included by kubeadm 1.31.
+- Released 3 patch versions for etcd stable releases between September - December 2023 .
+- Fixed an issue with our experimental distributed tracing feature to ensure sampling rates could be properly configured.
+
 
 2. Are there any areas and/or subprojects that your group needs help with (e.g. fewer than 2 active OWNERS)?
 
-<!--
-   Note: This list is generated from the KEP metadata in kubernetes/enhancements repository.
-      If you find any discrepancy in the generated list here, please check the KEP metadata.
-      Please raise an issue in kubernetes/community, if the KEP metadata is correct but the generated list is incorrect.
--->
+- An emerging risk for SIG etcd is the raft subproject which is currently operating on two primary approvers after a longstanding maintainer retired in July 2023. The two primary approvers have limited bandwidth for raft as they are also key maintainers for etcd. Over 2024 we need help growing a new maintainership for `etcd-io/raft`.
+
+- The second area of concern for SIG etcd is the etcd grpc-proxy which also lacks maintainership as well as evidence for use cases. This is an area of functionality we may consider for deprecation in future if we aren't able to overcome the the maintainership challenge through a subproject lead.
+
 
 3. Did you have community-wide updates in 2023 (e.g. KubeCon talks)?
 
-<!--
-  Examples include links to email, slides, or recordings.
--->
+The etcd project team has had talks at both Kubecon events in 2023:
 
-4. KEP work in 2023 (v1.27, v1.28, v1.29):
+- [KubeCon Europe 2023, On the hunt for etcd data inconsistencies](https://www.youtube.com/watch?v=IIMs0EjQZHg)
+- [KubeCon North America 2023, Forging a stronger bond between etcd and Kubernetes](https://www.youtube.com/watch?v=6JYgBJAjpNQ)
+- [KubeCon North America 2023, Secrets of running etcd](https://www.youtube.com/watch?v=aJVMWcVZOPQ)
 
+Additionally we held ContribFest events at both KubeCons in 2023 which have been successful in growing new contributors. In 2023 we also introduced a new fortnightly triage meeting to complement our existing fortnightly community meeting.
+
+All meeting recording and maintainer track talks are available on YouTube: https://www.youtube.com/@etcdio
+
```

Comment:
Should we mention the robustness test regular meeting too?

Nice to have so much things happening that we need to keep updating our reports.

Reply:
Good question - We definitely need to highlight this however it would go into the 2024 calendar year report as the meetings didn't start until this calendar year.  This pr only covers 2023.

Reply:
Ooh, right!